translations/ko.po

```diff
@@ -710,3 +745,8 @@ msgstr "YGOPRODECK 카드 데이터베이스에서 검색합니다."
 msgctxt "command-description"
 msgid "Search the Yugipedia wiki for a page and link to it."
 msgstr "Yugipedia에서 검색하고 링크를 출력합니다."
+
+#: src\commands\help.ts:50
+msgctxt "command-description"
+msgid "Get help with a Slash Command."
+msgstr ""
```

Comment:
I've been manually fixing the trailing newline, but I think we can also fix this by hard-forcing the gettext-parser module to resolve to version 6, since I already patched the dependency and ttag maintainers are occupied.